packages/core/src/BackgroundImage/BackgroundImage.js

```diff
@@ -31,8 +29,12 @@ BackgroundImage.defaultProps = {
 }
 
 BackgroundImage.propTypes = {
+  height: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.string,
+    PropTypes.array
+  ]),
```

Comment:
Could we replace this with `...height.propTypes`? Probably wouldn't hurt to do the same for `width`.

https://github.com/priceline/design-system/blob/7096fa92411261a21adf0c11c911d9e4a178c51e/packages/core/src/Text/Text.js#L91